lib/commits_loader.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+require 'active_support'
+require 'active_support/core_ext/numeric'
+require 'active_support/core_ext/time'
+require 'json'
+require 'tempfile'
+require 'csv'
+require 'zlib'
+require 'pg'
+require_relative './github'
+require_relative './aws_helper'
+
+class CommitsLoader
+  HEADERS = %i[object_id repository message_headline message author_name authored_date committed_date pushed_date]
+  BUCKET = 'artsy-data'
+
+  def self.load_recent_commits
+    new.load_recent_commits
+  end
+
+  def initialize(since: 5.days.ago, _until: nil, org: "artsy")
```

Comment:
Could this potentially create gaps (until we automate this) if we don't manually run within 5 days?

Reply:
Yes there could be gaps. Imagine if I'm working on a PR for a month, then do a traditional merge into `main`. Some of those commits will appear in the distant past of `git log`, and so won't be imported by this due to how Github's `since` and `until` search parameters operate.


We might want to expand this significantly, like to 30 days, and run it daily. What do you think?

Reply:
Yeah, that seems a reasonable start, and we can adjust up or down if we run into issues.

Reply:
Extending it a bit makes sense as Christmas and Thanksgiving create situations where multiple/all team members will be off for more than 5 days. Should eventual automation include a freshness/health metric or is it sufficient to see the latest commit in the dataset be recent enough?